BaragonAgentService/src/main/java/com/hubspot/baragon/agent/config/LoadBalancerConfiguration.java

```diff
@@ -35,6 +35,10 @@
   @NotNull
   private Set<String> domains = Collections.emptySet();
 
+  private int maxReloadsPerMinute = 0;
```

Comment:
i'd either make this `Optional<Integer>` or add an additional `bool rateLimitReloads`. Either way, we should probably default rate limiting to disabled for now and opt ourselves in for it.